vms/platformvm/state/camino_diff.go

```diff
@@ -287,6 +296,13 @@ func (d *diff) GetDeferredStakerIterator() (StakerIterator, error) {
 	return d.caminoDiff.deferredStakerDiffs.GetStakerIterator(parentIterator), nil
 }
 
+func (d *diff) SetValidatorUptime(subnetID ids.ID, nodeID ids.NodeID, uptime time.Duration) {
```

Comment:
currently, all camino diff maps are initialized in newCaminoDiff
I would suggest either to change all maps inits to lazy init like its done here, or change this place so it will be like with other maps